src/loaders/gtdb/gtdb_genome_attribs_loader.py

```diff
@@ -18,62 +15,42 @@
-NOTE: the function name should be exactly the same as the non-existing feature name.
+NOTE: The Document Key ("_key") in Arango DB is generated by applying a hash function to a string consisting of the 
+kbase_collection, load_version, and genome id parsed from the accession field from the GTDB metadata file. 
+(e.g. GTDB_r207.kbase.1_GCA_000016605.1, GTDB_r207.kbase.1_GCA_000169995.1)
 
-Any features that are used for computing the non-existing features should be added to HELPER_FEATURES. They will be
-eventually dropped from the result dataframe.
+usage: gtdb_genome_attribs_loader.py [-h] --load_ver LOAD_VER [--kbase_collection KBASE_COLLECTION] [-o OUTPUT]
+                                     load_files [load_files ...]
 
+PROTOTYPE - Prepare GTDB genome statistics data in JSON format for arango import.
 
-usage: gtdb_genome_attribs_loader.py [-h]
-                                   --load_version
-                                   LOAD_VERSION
-                                   [--kbase_collection KBASE_COLLECTION]
-                                   [-o OUTPUT]
-                                   load_files
-                                   [load_files ...]
+options:
+  -h, --help            show this help message and exit
 
+required named arguments:
+  load_files            GTDB genome metadata files
+  --load_ver LOAD_VER   KBase load version. (e.g. r207.kbase.1)
 
-e.g. gtdb_genome_attribs_loader.py bac120_metadata_r207.tsv ar53_metadata_r207.tsv --load_version 207
-     gtdb_genome_attribs_loader.py bac120_metadata_r207.tsv ar53_metadata_r207.tsv --load_version 207 --kbase_collection gtdb
-     gtdb_genome_attribs_loader.py bac120_metadata_r207.tsv ar53_metadata_r207.tsv --load_version 207 --kbase_collection gtdb --output  gtdb_genome_attribs.json
-"""
+optional arguments:
+  --kbase_collection KBASE_COLLECTION
+                        kbase collection identifier name (default: GTDB)
+  -o OUTPUT, --output OUTPUT
+                        output JSON file path (default: gtdb_genome_attribs.json
 
-EXIST_FEATURES = helper.EXIST_FEATURES
-NON_EXIST_FEATURES = {'high_checkm_marker_count'}  # genome statistics to be computed
-HELPER_FEATURES = {'checkm_marker_count'}  # additional features needed to compute statistics
+
+e.g. gtdb_genome_attribs_loader.py bac120_metadata_r207.tsv ar53_metadata_r207.tsv --load_version r207.kbase.1
+     gtdb_genome_attribs_loader.py bac120_metadata_r207.tsv ar53_metadata_r207.tsv --load_version r207.kbase.1 --kbase_collection GTDB
+     gtdb_genome_attribs_loader.py bac120_metadata_r207.tsv ar53_metadata_r207.tsv --load_version r207.kbase.1 --kbase_collection GTDB --output  gtdb_genome_attribs.json
+"""
 
 # change header to specific context (default mapper: capitalize first character)
 HEADER_MAPPER = {'accession': names.FLD_GENOME_ATTRIBS_GENOME_NAME}
 
 
-def _compute_attribs(df, computations):
```

Comment:
moved the methods responsible for computing genome attributes to the `compute_genome_attribs.py` script

Reply:
*removed. (Additional genome attributes are generated by running other tools (checkm2 and gtdb-tk) rather than being derived from the metadata file provided by GTDB)